### PR TITLE
MicrosoftApiModule - added  resource usage to _oproxy_authorize

### PR DESCRIPTION
**Pull request by tallieber**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [39619](https://github.com/demisto/etc/issues/39619)

## Description
Azure log analytics uses MicrosoftApiModule client as multi_resource. Added the resource parameter to the post request.

## Minimum version of Cortex XSOAR
- [x] 5.5.0
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by tallieber** [comment]
@bakatzir  
I need I think to ignore this error on validation: `Packs/ApiModules/Scripts/MicrosoftApiModule/MicrosoftApiModule.yml: [CJ105] - All test playbooks for MicrosoftApiModule in this pack are currently skipped. Please unskip at least one of the relevant test playbooks.`

ApiModule should not have test playbook.

**Comment by tallieber** [comment]
@dantavori If you can review this PR, this change is related to AzureLogAnalytics pack, when we are using MicrosoftApiModulw with multi resource.

**Review by dantavori — COMMENTED** [review]
looks good to me, let's make sure that:
1. Azure Log Analytics integration authenticates to both resources by calling two commands that use different APIs:
   - `!azure-log-analytics-execute-query query="Usage | take 10"`
   - `azure-log-analytics-list-saved-searches`
2. Any non multi-resource integration (e.g. Microsoft ATP) is not affected from this change.

**Comment by tallieber** [comment]
> looks good to me, let's make sure that:
> 
> 1. Azure Log Analytics integration authenticates to both resources by calling two commands that use different APIs:
>    
>    * `!azure-log-analytics-execute-query query="Usage | take 10"`
>    * `azure-log-analytics-list-saved-searches`
> 2. Any non multi-resource integration (e.g. Microsoft ATP) is not affected from this change.

Hey, 1. checked that it works fine, 2. checked Microsoft ATP, the test-module and few list command works fine

**Review by dantavori — APPROVED** [review]

File: Packs/AzureLogAnalytics/ReleaseNotes/1_0_10.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### Azure Log Analytics (Beta)
+- Internal code improvements.
```
Comment (bakatzir) [inline Packs/AzureLogAnalytics/ReleaseNotes/1_0_10.md:4]:
```suggestion
- Fixed an issue in which not all of the resources had been associated with the access token.
```

**Review by bakatzir — APPROVED** [review]

**Comment by bakatzir** [comment]
Force merging as it passed a few servers, due to gitlab outage.

**Pull request merged**